`gcloud app versions` sometimes doesn't include the newly deployed version

e.g. https://travis-ci.org/web-platform-tests/wpt.fyi/jobs/384903195

```
ERROR: (gcloud.app.versions.describe) NOT_FOUND: Version "perf" does not exist.
Makefile:173: recipe for target 'deploy_staging' failed
make: *** [deploy_staging] Error 1
```

There might be a race condition. Depending on how often this happens, we can ignore this edge case, add a `sleep`, or do something more involved. cc @lukebjerring

--- a/util/deploy.sh
+++ b/util/deploy.sh
@@ -82,8 +82,4 @@ set -e
 if [[ -z "${QUIET}" ]]; then info "Executing..."; fi
 ${COMMAND} || fatal "Deploy returned non-zero exit code $?"
 
-# Comment on the PR if running from Travis.
-DEPLOYED_URL=$(gcloud app versions describe ${VERSION} -s default | grep -Po 'versionUrl: \K.*$')
-echo "Deployed to ${DEPLOYED_URL}"
-
 exit 0

--- a/util/travis-deploy-staging.sh
+++ b/util/travis-deploy-staging.sh
@@ -50,7 +50,7 @@ docker exec -t -u $(id -u $USER):$(id -g $USER) "${DOCKER_INSTANCE}" \
         BRANCH_NAME="${TRAVIS_PULL_REQUEST_BRANCH:-$TRAVIS_BRANCH}" 2>&1 \
             | tee ${TEMP_FILE}
 if [ "${EXIT_CODE:=${PIPESTATUS[0]}}" != "0" ]; then exit ${EXIT_CODE}; fi
-DEPLOYED_URL="$(grep -Po 'Deployed to \K[^\s]+' ${TEMP_FILE} | tr -d '\n')"
+DEPLOYED_URL="$(grep 'Deployed.*to' ${TEMP_FILE} | sed -e 's/Deployed.*to \[\(.*\)\]/\1/')"
 
 # Add a GitHub comment to the PR (if there is a PR).
 if [[ -n "${TRAVIS_PULL_REQUEST_BRANCH}" ]];

